schema.sql

```diff
@@ -27,6 +27,16 @@ CREATE TABLE saved_prop(
 
 )
 
-CREATE TABLE propertoes(
-  id BIGINT NOT NULL UNIQUE
+CREATE TABLE properties(
+  id BIGINT NOT NULL UNIQUE,
+  photo_url VARCHAR(300),
+  descrip VARCHAR(300),
```

Comment:
If the descriptions are going to be of a realistic size, 300 might be too small?

Reply:
![Screen Shot 2020-10-16 at 9 13 02 AM](https://user-images.githubusercontent.com/36867262/96282683-ddbfb300-0f8f-11eb-9040-bb0b9ee0e072.png)
the description used in app is actually the listing title for this component. It should be fairly short (< 100 chars)

Reply:
Ah, OK, thought the description was the long, paragraph length area.